.github/CODEOWNERS

```diff
@@ -1,5 +1,13 @@
 # Global Reviewer
-* @rancher/mapps
+docs @rancher/mapps
+scripts @rancher/mapps
+CNAME @rancher/mapps
+Makefile @rancher/mapps
+README.md @rancher/mapps
+_config.yml @rancher/mapps
+configuration.yaml @rancher/mapps
+regsync.yaml @rancher/mapps
+index.html
```

Comment:
This also should be reviewed by the Mapps team

Reply:
@lucasmlp Got catch!